requirements/base.txt

```diff
@@ -1,5 +1,4 @@
 asttokens>=2.0.4
 click
-pyseeyou==1.0.0
-pytz
-requests==2.22.0
+pyseeyou==1.0.2
+requests==2.31.0
```

Comment:
It looks like requests 2.31 doesn't support Python 3.6, probably we should drop Python 3.6 from the CI.